### PR TITLE
Add support for prefunded_accounts during env creation

### PR DESCRIPTION
**Pull request by vdamle**
Adds support to configure pre-funded accounts with ether balances
during environment creation

### CODE REVIEW

File: cmd/environment.go [hunk 1]
```diff
@@ -77,7 +77,15 @@ var environmentCreateCmd = &cobra.Command{
 		validateConsortiumID("environment")
 
 		client := getNewClient()
-		environment := kld.NewEnvironment(name, desc, provider, consensus, multiRegion, blockPeriod)
+		accountBalances := map[string]string{}
```
Comment (jimthematrix) [inline cmd/environment.go:80]: can I suggest using a single param to accomplish this:
`--pre-fund ["0xf601c8a58a738c1055094d0cf3018266d562c4a5:1000","0xb68965e17da0A5602A7721053accdB83067726f0:2000000"]`

**Comment by vdamle** [comment]
results from testing this with a local build:

```
 ✘  kaleido-sdk-go   master ●  ./kld create environment --api-url <url> --verbose 1 -c zzputzf9rt -a 'f601c8a58a738c1055094d0cf3018266d562c4a5' -a 'e723c8a58a738c1055094d0cf3018266d562c3a4' -n 'kld-go-env' -b '10000000000' -b '3000000000'

{"name":"kld-go-env","description":"","provider":"quorum","consensus_type":"raft","block_period":0,"prefunded_accounts":{"e723c8a58a738c1055094d0cf3018266d562c3a4":{"balance":"3000000000"},"f601c8a58a738c1055094d0cf3018266d562c4a5":{"balance":"10000000000"}},"limits":{"nodes":30},"autopause_init_delay":168,"autopause_idle_hours":72,"_id":"zzhubqqqve","chain_id":1907950444,"node_list":[],"service_list":[],"limit_initial_signers":false,"test_features":{"multi_region":false,"admin_override":false,"hybrid":false,"enable_constantinople":true},"state":"initializing","release_id":"zzan3q3clh","_revision":"0","created_at":"2020-10-14T14:20:39.838Z","consortia_id":"zzputzf9rt"}
```

**Review by jimthematrix — CHANGES_REQUESTED** [review]
Looks good, have a suggestion for alternative parameter name and format, for your consideration

File: cmd/environment.go (hunk 1 above)
Comment (jimthematrix) [reply cmd/environment.go:80]: sounds good, can go in as-is then.

**Pull request merged**